test_data/samples_test.go

```diff
@@ -322,46 +323,6 @@ func TestMarkdown(t *testing.T) {
 	})
 }
 
-func TestBincapzIgnored(t *testing.T) {
```

Comment:
This test took ~35 seconds only to prove that we don't get any output. With the name change, this will fail so it makes sense to remove it for now (we'd need a `malcontent` sample for the samples repository anyway).